### PR TITLE
Services: Stubbed more services.

### PR DESCRIPTION
**Pull request by Subv**
The new service stubs are:
- am:sys
- boss:P
- cecd:s
- frd:a
- gsp::Lcd
- news:s
- ns:s

Implemented FSUser::CreateExtSaveData.

### CODE REVIEW

**Comment by linkmauve** [comment]
There is a typo in the second commit’s name, s/cmale/cmake/

File: src/core/hle/service/fs/archive.cpp [hunk 1]
```diff
@@ -432,6 +432,27 @@ ResultCode FormatSaveData() {
...
+    for (unsigned i = 0; i < 4; ++i)
+        binary_path.push_back((low >> (8 * i)) & 0xFF);
+    // Next is the high word
+    for (unsigned i = 0; i < 4; ++i)
+        binary_path.push_back((high >> i) & 0xFF);
+    FileSys::Path path(binary_path);
+    std::string nand_directory = FileUtil::GetUserPath(D_NAND_IDX);
+    std::string base_path = FileSys::GetExtDataContainerPath(nand_directory, true);
+    std::string extsavedata_path = FileSys::GetExtSaveDataPath(base_path, path);
+    FileUtil::CreateFullPath(extsavedata_path);
```
Comment (linkmauve) [inline src/core/hle/service/fs/archive.cpp]: Return RESULT_SUCCESS only if this function returned true, and an error otherwise.

**Comment by neobrain** [comment]
LGTM

**Comment by archshift** [comment]
LGTM after @linkmauve's comment is addressed

**Comment by bunnei** [comment]
@Subv ping please fix `CMale` :) 
And now rebase...

**Comment by Subv** [comment]
Should be ready

**Comment by bunnei** [comment]
LGTM

**Pull request merged**